init_distro_rid.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Please consider using the POSIX variant of new scripts: [raw gist](https://gist.githubusercontent.com/am11/e11d49bc924cde96b04dea23bed5ed4c/raw/19dfb0088702cc5a8fcba494110aa06aa80ad9fa/init_distro_rid.sh) | [diff](https://gist.github.com/am11/e11d49bc924cde96b04dea23bed5ed4c/revisions)

converted manually with the helps/tips from https://www.shellcheck.net/

/reduce-bashism :)

Reply:
I think my preference is to tackle this another day.  @sbomer @echesakovMSFT @janvorli @RussKeldorph opinions?